src/app/preview-warning/preview-warning.component.html

```diff
@@ -1,7 +1,17 @@
-<mat-card appearance="outlined" class="alert alert-warning w-100" data-cy="preview-warning">
+<mat-card *ngIf="!isBubble" appearance="outlined" class="alert alert-warning w-100 mb-0" data-cy="preview-warning">
   <mat-card-content>
     <mat-icon class="alert-warning-icon">warning</mat-icon>
     <strong> Notice:</strong> {{ featureName }} is an early access feature. Consider this a prototype that may be more likely to have
```

Comment:
Worth making/possible to make this String a readonly variable in the service?

Reply:
Yeah it's doable and a good idea since it's also the bubble's tooltip, will implement